package-overrides/npm/redbox-react@1.1.1.json

```diff
@@ -0,0 +1,9 @@
+{
+  "registry": "npm",
+  "main": "dist/redbox",
```

Comment:
I think `dist/redbox` might have been from some earlier configs circling around where Browserify/Webpack build had issues. I believe those were fixed since then but I might be wrong.